docs/sql/statements/create-user.rst

```diff
@@ -24,14 +25,16 @@ Description
 ===========
 
 ``CREATE USER`` is a management statement to create a new database user in the
-CrateDB cluster. The newly created user does not have any special privileges.
+CrateDB cluster. The newly created user does not have any special privileges,
+and those must be assigned afterwards, for details see the
+:ref:`privileges documentation<administration-privileges>`.
 The created user can be used to authenticate against CrateDB, see
 :ref:`admin_hba`.
 
 The statement allows to specify a password for this account. This is not
 necessary if password authentication is disabled.
 
-For usage of the ``CREATE USER`` statement see
+For usages of the ``CREATE USER`` statement see
 :ref:`administration_user_management`.
```

Comment:
I don't want to add it now, because to make sense I would need to expose LOGIN flag which we don't as currently we don't allow this to be modifying, (a user cannot become a role or vice versa).
What do you think?